Samples/UIWidgetsSamples_2019_4/Assets/UIWidgetsExample.cs

```diff
@@ -44,9 +28,9 @@ public static void LogMessageFromCpp(IntPtr message, int iSize)
             // FontManager.instance.addFont(Resources.Load<Font>(path: "path to material icons"), "Material Icons");
 
             // Pass the C# Debug function reference to C++ to save, and then C++ calls C# through the function pointer
-             UIWidgetsExample.InitCSharpDelegate(UIWidgetsExample.LogMessageFromCpp);
-
-             base.OnEnable();
+            NativeConsole.InitLogMessageFromCppDelegate(NativeConsole.LogMessageFromCpp);
```

Comment:
move this call to a proper place in UIWidgetsPanel.cs, maybe OnEnable?
we should make the details of native console logging invisible to users